install/disconnected_install.adoc

```diff
@@ -578,247 +597,42 @@ name=rhel-7-server-ansible-2.6-rpms
-installation, the image streams are created in the *openshift* project by
-loading data from the *_/usr/share/openshift/examples_* directory.
-
-.. Delete the existing image streams:
-+
-[source, bash]
-----
-$ oc delete is -n openshift --all
-----
-
-.. Back up the files in the *_/usr/share/openshift/examples/_* directory.
-
-.. In the *_image-streams-rhel7.json_* file in the
-*_/usr/share/openshift/examples/image-streams_* directory, set the image `name`
-in the `spec` stanza for each builder image stream to reference your internal
-Docker registry.
-+
-Use the following format for each `name`:
-+
-----
-<registry_ip>:5000/openshift/<image_name>
-----
-+
-Be sure to use the correct IP address for your registry IP, and note that the
-repository name is *openshift*.
-+
-For example, change:
-+
-----
-"from": {
-  "kind": "DockerImage",
-  "name": "registry.access.redhat.com/rhscl/httpd-24-rhel7"
-}
-----
-+
-to:
-+
-----
-"from": {
-  "kind": "DockerImage",
-  "name": "172.30.69.44:5000/openshift/httpd-24-rhel7"
-}
-----
-
-.. Set the image `name` in the `spec` stanza for each JBoss image stream in the
-*_/usr/share/openshift/examples/xpaas-streams/jboss-image-streams.json_* file
-to reference your internal Docker registry.
-
-. Load the container images:
-
-.. Log in to the Docker registry using the token for user that you created and 
-the registry service IP address:
-+
-[source, bash]
-----
-$ docker login -u adminuser -e mailto:adminuser@abc.com \
-   -p $MYTOKEN $REGISTRY:5000
-----
-
-.. Push the Docker images:
-+
-[source, bash]
-----
-$ docker push $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:1.1
-$ docker push $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:1.2
-$ docker push $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:latest
-----
-
-.. Load the updated image stream definitions:
-+
-[source, bash]
-----
-$ oc create -f /usr/share/openshift/examples/image-streams/image-streams-rhel7.json -n openshift
-$ oc create -f /usr/share/openshift/examples/xpaas-streams/jboss-image-streams.json -n openshift
-----
-
-.. Verify that all the image streams use the correct tags:
+. xref:running_install.adoc#install-running-installation-playbooks[Run the
+installation playbooks].
 +
-[source, bash]
-----
-$ oc get imagestreams -n openshift
-NAME                                 DOCKER REPO                                                      TAGS                                     UPDATED
-jboss-webserver30-tomcat7-openshift  $REGISTRY/jboss-webserver-3/webserver30-jboss-tomcat7-openshift  1.1,1.1-2,1.1-6 + 2 more...              2 weeks ago
-...
-----
-
-////
-[[disconnected-installing-a-router]]
-== Installing a Router
-
-At this point, the {product-title} environment is almost ready for use. It is
-likely that you will want to
-xref:../install_config/router/index.adoc#install-config-router-overview[install and configure a
-router].
-////
+[IMPORTANT]
+====
+You must provide the value of the `<tag>` for the images that you
```

Comment:
Here I think "must" is not accurate. Because in the above section, user tag one image with both minor version and major version, ideally not have to set `openshift_image_tag`.